casper/src/test/scala/coop/rchain/casper/helper/BlockGenerator.scala

```diff
@@ -1,22 +1,40 @@
-package coop.rchain.casper
+package coop.rchain.casper.helper
 
+import cats._
+import cats.data.StateT
+import cats.implicits._
+import cats.mtl.MonadState
 import com.google.protobuf.ByteString
-import coop.rchain.casper.BlockDagState._
+import coop.rchain.casper.BlockDag
+import coop.rchain.casper.Estimator.{BlockHash, Validator}
+import coop.rchain.casper.helper.BlockGenerator.BlockDagState
+import coop.rchain.casper.protocol._
 import coop.rchain.casper.util.ProtoUtil
+import coop.rchain.catscontrib.Capture
 import coop.rchain.crypto.hash.Blake2b256
+import coop.rchain.p2p.EffectsTestInstances.LogicalTime
+import coop.rchain.shared.Time
 
 import scala.collection.immutable.{HashMap, HashSet}
 import scala.language.higherKinds
-import coop.rchain.catscontrib._
-import Catscontrib._
-import cats._
-import cats.data._
-import cats.implicits._
-import coop.rchain.casper.Estimator.{BlockHash, Validator}
-import coop.rchain.casper.protocol._
+
+object BlockGenerator {
+  implicit def idCapture: Capture[Id] = new Capture[Id] {
```

Comment:
There's a `Capture[Id]` instance in the `catscontrib` right?

Reply:
As I posted on the #scala channel, I got the following error when I tried to import that

```
26: ambiguous implicit values:
[error]  both method idCapture in trait CaptureInstances of type => coop.rchain.catscontrib.Capture[cats.Id]
[error]  and method idCapture in trait CaptureInstances of type => coop.rchain.catscontrib.Capture[cats.Id]
[error]  match expected type coop.rchain.catscontrib.Capture[cats.Id]
[error]   implicit val timeEff = new LogicalTime[Id]
```

Reply:
I think if you `import coop.rchain.catscontrib._` then scala figures it out properly.

Reply:
@birchmd Yup that did the trick!